### DISCUSSION TITLE
Markdown TeX rendering errors

### DISCUSSION THREAD

**Discussion by MattPChoy** [community/community#28623] [Discussions]
I have the following TeX:

$$
\begin{aligned}
\sum^{m+1}_{k=1} (ca_k+b_k) &= (c a_{m+1} + b_{m+1}) + \sum^m_{k=1} (ca_k + b_k)\\
\end{aligned}
$$
$$
\begin{aligned}
\sum_{k=1}^{m+1} (ca_k+b_k) &= (c a_{m+1} + b_{m+1}) + \sum^m_{k=1} (ca_k + b_k)\\
\end{aligned}
$$

Why is the first $a_{m+1}$ term after the first equals sign being rendered as $am+1$?

**Comment by MattPChoy** [comment]
Also just noticed that the subscript $k=1$ for the summation is broken

**Comment by airtower-luna** [comment]
Could you also post the TeX code without rendering (e.g. in a plain code block)? It's hard to guess what might be going on without seeing it.

**Reply by MattPChoy** [reply]
@airtower-luna \sum^{m+1}_{k=1} (ca_k+b_k) &= (c a_{m+1} + b_{m+1}) + \sum^m_{k=1} (ca_k + b_k)

**Reply by airtower-luna** [reply]
You forgot the code block around the TeX, I had to figure out where the underscores that Markdown interpreted as emphasis were. :confused: 

Anyway, looks like an odd bug. If I switch the order of the subscript and superscript in the code on the first `\sum` around it works:

```
$$\sum_{k=1}^{m+1} (ca_k+b_k) = (c a_{m+1} + b_{m+1}) + \sum_{k=1}^m (ca_k + b_k)$$
```

You might want to [send a bug report to GitHub support](https://support.github.com/contact/bug-report).
